src/featurizers/mitie_featurizer.py

```diff
@@ -10,13 +10,11 @@
 class MITIEFeaturizer(object):
 
     def __init__(self, fe_file):
-        if not os.path.isfile(fe_file):
-            self.download(fe_file)
-            return  # need a cleaner solution for this
         self.feature_extractor = total_word_feature_extractor(fe_file)
         self.ndim = self.feature_extractor.num_dimensions
 
-    def download_fe_file(self, fe_file):
+    @staticmethod
+    def download_fe_file(fe_file):
```

Comment:
we could move this to `utils/mitie.py` & let it be invoked through command line, similar to spacy sth like `python -m rasa_nlu.mitie.en.download`